work9/index.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+const ul = document.getElementById("js-list");
+const obj = [{
+  to: "bookmark.html",
+  img: "1.png",
+  alt: "画像1",
+  text: "ブックマーク",
+},
+{
+  to: "message.html",
+  img: "2.png",
+  alt: "画像2",
+  text: "メッセージ"
+},
+]
+
+function getObject() {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve(obj);
+
+      reject(new Error("エラー"));
+
+    }, 3000);
+  })
+}
+
+async function createElement() {
+  const fragment = document.createDocumentFragment();
+
+  const getObj = await getObject();
+
+  ul.style.backgroundImage = "none";
+
+  getObj.forEach((item) => {
+    const li = document.createElement("li");
+    const a = document.createElement("a");
+    const img = document.createElement("img");
+
+    a.href = item.to;
+    a.textContent = item.text;
+    img.src = item.img;
+    img.alt = item.alt;
+
+    fragment.appendChild(li).appendChild(a).prepend(img);
+  });
+  ul.appendChild(fragment);
+}
+
+createElement();
```

Comment:
I think it's better after the loading.

Reply:
Thank you.

I couldn't understand this task of work 9.
Order,
1, "loading"
2, get an array
3, When I get the array, hide "loading" and create an element

right?

Please confirm that the correction has been completed.

d473d59

Reply:
You’re right.
They will be executed in order from the top.
So,I think the corrected version is more natural.

Even before the correction, it still works because it is an asynchronous process.

Reply:
> Even before the correction, it still works because it is an asynchronous process.

I didn't understand the meaning of work9, so I think I didn't understand aysnc await and promises.

Please confirm that the correction has been completed.

f53ee5b